apps/docs/publisher_agent.md

```diff
@@ -17,19 +17,18 @@ Here we open a websocket at `ws://localhost:5216/publish` to receive price updat
 }
 ```
 
-Once you've gotten a StorkAuth key from Stork, you can use the [generate_keys.py](../../python/src/generate_keys/generate_keys.py) script to generate your EVM and/or Stork keys and to build the `keys.json` file. Make sure not to check this file into version control or share it in any way. 
+You can use the [generate_keys.py](../../python/src/generate_keys/generate_keys.py) script to generate your EVM and/or Stork keys and to build the `keys.json` file. Make sure not to check this file into version control or share it in any way. 
 
 You can also generate your own keys and build your own `keys.json` file - it follows the structure of the [Keys struct](../lib/publisher_agent/config.go):
 ```json
 {
   "EvmPrivateKey": "0x8b558d5fc31eb64bb51d44b4b28658180e96764d5d5ac68e6d124f86f576d9de",
   "EvmPublicKey": "0x99e295e85cb07c16b7bb62a44df532a7f2620237",
-  "OracleId": "oracl",
-  "StorkAuth": "fake_auth"
```

Comment:
remove all references to StorkAuth from this readme since publishers will no longer use it